templates/Dockerfile_ui2.template

```diff
@@ -2,33 +2,16 @@ FROM ubuntu:18.04 as builder
 
 RUN apt update \
     && apt install -yq \
-    openjdk-11-jre \
-    git \
     wget \
     && apt clean
 
-# install nvm
-RUN wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh | bash
-
-# cache busting in order to get the up-to-date Git repo
-RUN touch {{ DOCKSTORE_UI2_VERSION }}.out
-
-RUN git clone https://github.com/dockstore/dockstore-ui2.git
-WORKDIR /dockstore-ui2
-RUN git checkout {{ DOCKSTORE_UI2_VERSION }}
-
-# install node version from .nvmrc file
-run bash -i -c 'nvm install'
-
-# inherit $CI in order to suppress Cypress TTY output when running on CI server
-ARG CI
-ENV CI $CI
-RUN bash -i -c 'npm ci'
-
-# build
-RUN bash -i -c 'NODE_OPTIONS="--max-old-space-size=4096" npm run build.prod'
+# Download index.html for UI we will be serving
+RUN wget -qO- https://gui.dockstore.org/{{ DOCKSTORE_UI2_VERSION }}/index.html
 
+# Insert full qualified paths.
```

Comment:
what are the pros/cons of using sed in this way as opposed to the two options in https://github.com/angular/angular-cli/issues/3557
1) Using `ng build --prod --base-href https://gui.dockstore.org//{{ git reference }}` in the circle ci build
2) Setting deployUrl in the build as per https://github.com/angular/angular-cli/issues/3557#issuecomment-433868214

Reply:
I think the major con is it takes us back to having separate builds for production and staging.

Also, I'm not sure if the base href doesn't affect ajax calls, but I would need to look into it.

Reply:
Hmmm, don't follow.  
 `https://gui.dockstore.org//{{ git reference }}` seems the same between staging and prod, but I could be missing something